src/Aquifer.Data/EventHandlers/ResourceStatusChangeHandler.cs

```diff
@@ -22,6 +22,7 @@ public static async Task HandleAsync(DbContextOptions<AquiferDbContext> dbContex
             .ToList()
             .ForEach(x =>
             {
+#pragma warning disable IDE0010 // Add missing cases to switch statement
```

Comment:
Is this something we can just disable globally? Would we ever care about this if the functionality didn't need it?

Reply:
I think it's better to explicitly ignore the warning when we are sure it's not needed.  Example:
1. We write code that handles all enum values in a switch statement.
2. Someone adds a new enum value.  The existing switch statement is not updated to handle it.
3. The code explodes at runtime or doesn't have correct execution.

This warning forces you to explicitly add handling for the new enum value.  If we wanted to we could handling here instead of disabling the warning by including all of the other values with an empty case statement (but an empty case statement also can give warnings).